en/docs/learn/examples/proxy-service-examples/securing-proxy-services.md

```diff
@@ -44,9 +45,8 @@ This sample security policy file validates username token and admin role is allo
 
 Create the artifacts:
 
-1. [Set up WSO2 Integration Studio]({{base_path}}/develop/installing-wso2-integration-studio).
-2. [Create an integration project]({{base_path}}/develop/create-integration-project) with an <b>ESB Configs</b> module and an <b>Composite Exporter</b>.
-3. Create the [proxy service]({{base_path}}/develop/creating-artifacts/creating-a-proxy-service) and [security policy]({{base_path}}/develop/creating-artifacts/registry/creating-local-registry-entries) with the configurations given above.
+{!includes/build-and-run.md!}
+3. Create the [proxy service]({{base_path}}/develop/creating-artifacts/creating-a-proxy-service), [security policy]({{base_path}}/develop/creating-artifacts/registry/creating-local-registry-entries) and [endpoint]({{base_path}}/develop/creating-artifacts/creating-endpoints) with the configurations given above.
 4. [Deploy the artifacts]({{base_path}}/develop/deploy-artifacts) in your Micro Integrator.
 
 Set up the back-end service:
```

Comment:
Create a new SOAP project in the SoapUI using following wsdl file:

->

Create a new SOAP project in SoapUI using the following wsdl file: